Annex60/Fluid/Movers/SpeedControlled_Nrpm.mo

```diff
@@ -16,7 +16,7 @@ model SpeedControlled_Nrpm
   parameter Real speed(final unit="1/min") = 0
     "Speed set point when using constant set point"
     annotation(Dialog(enable=inputType == Annex60.Fluid.Types.InputType.Constant));
-  parameter Real[:] speeds(each final unit="1/min") = {0}
+  parameter Real[:] speeds(each final unit="1/min") = per.speeds
```

Comment:
I changed it to `speeds_rpm` to indicate that these speeds have non-SI units. Similarly to the situation where we use `T_degC`.